packages/libs/error-stack/Cargo.toml

```diff
@@ -1,9 +1,13 @@
+[workspace]
+members = [".", "macros"]
+default-members = ["."]
+
 [package]
 name = "error-stack"
 version = "0.1.1"
 authors = ["HASH"]
 edition = "2021"
-rust-version = "1.61.0"
+rust-version = "1.63.0"
```

Comment:
Do you mean to implicitly update this as part of this PR?

Reply:
This was already done in https://github.com/hashintel/hash/pull/944 and https://github.com/hashintel/hash/pull/969, I just forgot that this is also captured in `Cargo.toml`.